schedule/install/11sp4_s390_install.yaml

```diff
@@ -8,6 +8,7 @@ schedule:
   - installation/disk_activation
   - installation/partitioning_finish
   - installation/installer_timezone
+  - installation/install_11SP4_preparation
```

Comment:
please use lower case only, also see "install_11sp4_configuration" further down which is also lower case so better be consistent

Reply:
Yes, updated. Thanks.